librz/core/cmd_debug.c

```diff
@@ -1443,376 +1357,462 @@ static bool get_bin_info(RzCore *core, const char *file, ut64 baseaddr, PJ *pj,
 	return true;
 }
 
-static int cmd_debug_map(RzCore *core, const char *input) {
+// dm
```

Comment:
I'm tempted to remove these comments... They will likely become wrong at some point. I'd prefer to focus on https://github.com/rizinorg/rizin/issues/691 to correctly find the right handler.